src/main/java/com/exactpro/th2/check1/configuration/Check1Configuration.java

```diff
@@ -34,6 +38,14 @@
     @JsonProperty(value="rule-execution-timeout", defaultValue = "5000")
     private long ruleExecutionTimeout = 5000L;
 
+    @JsonProperty(value="decimal-precision", defaultValue = "0.005")
+    private double decimalPrecision = 0.005;
+
+    @JsonProperty(value="time-precision", defaultValue = "PT3S")
+    @JsonDeserialize(using = DurationDeserializer.class)
```

Comment:
You can either register a JavaTimeModule in the mapper or use the existing deserialized `com.fasterxml.jackson.datatype.jsr310.deser.DurationDeserializer` from `com.fasterxml.jackson.datatype:jackson-datatype-jsr310`